src/main/antlr4/GobraParser.g4

```diff
@@ -35,10 +35,19 @@ maybeAddressableIdentifier: IDENTIFIER ADDR_MOD?;
 // Ghost members
 
 sourceFile:
-  packageClause eos (importDecl eos)* (
+  (initPost eos)* packageClause eos (importDecl eos)* (
     (specMember | declaration | ghostMember) eos
   )* EOF;
 
+initPost: INIT_POST expression;
+
+importPre: IMPORT_PRE expression;
+
+importSpec: (importPre eos)* alias = (DOT | IDENTIFIER)? importPath;
+
+importDecl: (importPre eos)* IMPORT importSpec
```

Comment:
the idea here is to support two patterns:
- the first one, shown below, is for when you import a single package:
```go
//@ importRequires P
import "pkg"
```
Here, the import requires may appear before the import clause per se
- the second one is for when you have a list of imports in a single import clause:
```go
import (
    //@ importRequires P1
    "pkg1"
    //@ importRequires P2
    "pkg2"
)
```

The change you suggest forces us to use the more awkward snippet
```go
import (
    //@ importRequires P
    "pkg"
)
```
instead of the first pattern I showed

Reply:
I agree but we also support this right now which is even more weird:
```
//@ importRequires P
import (
    //@ importRequires Q
    "pkg"
)
```
If I have the code still correctly in mind, I think `Q` gets simply dropped, i.e. replaced by P.

Reply:
Ah, I see what you mean. Actually, Gobra would not parse this snippet - basically, Gobra only parses the import precondition when after it there is an `import "PATHNAME"` pattern. However, I realize that this may not be super clear and that the parsing error may be confusing. I will make the error message clearer